test/leiningen/monolith_test.clj

```diff
@@ -0,0 +1,69 @@
+(ns leiningen.monolith-test
+  (:require
+    [clojure.java.io :as io]
+    [clojure.test :refer [deftest is]]
+    [lein-monolith.test-utils :refer [use-example-project read-example-project]]
+    [leiningen.monolith :as sut]))
+
+
+(use-example-project)
+
+
+(defn- absolute-path
+  "Return an absolute java.nio.file.Path for the given file-ish input."
+  [x]
+  (.. (io/as-file x) toPath toAbsolutePath))
+
+
+(defn- read-pprint-output
+  "Runs lein pprint with the given key path using the :monolith/all profile."
+  [& ks]
+  (->> (map str ks)
+       (apply vector "pprint")
+       (sut/with-all (read-example-project))
```

Comment:
I'm aware of the convention of using `sut` to mean, "system under test"; I think this would read better if the alias were `monolith`, which also matches Clojure's last-segment-of-namespace aliasing conventions:
```suggestion
       (monolith/with-all (read-example-project))
```